src/hotspot/share/runtime/globals.hpp

```diff
@@ -1101,7 +1101,7 @@ const int ObjectAlignmentInBytes = 8;
   notproduct(bool, CollectIndexSetStatistics, false,                        \
           "Collect information about IndexSets")                            \
                                                                             \
-  develop(intx, FastAllocateSizeLimit, 128*K,                               \
+  develop(int, FastAllocateSizeLimit, 128*K,                                \
```

Comment:
Ensure the value is either never printed or else the format specifier is updated accordingly.

Reply:
Okay.
It is only printed in `[Global flags]` printing.